### DISCUSSION TITLE
auto-hide popup after manual show/toggle

### DISCUSSION THREAD

**Discussion by tomoakley** [bennypowers/nvim-regexplainer#28] [Q&A] [answered]
Hi, love the plugin! Will make writing regex so much better. I'm wondering if this is either a bug or needs implementing; I want to have `auto = false` for the config, so I have to manually show the popup, but then when I move my cursor away on to a different line, it auto hides. Currently the popup persists until I hide it/toggle it off again. Not a deal breaker but more keystrokes than necessary!

**Comment by bennypowers** [comment, marked as answer]
would this do it?
```lua
vim.keymap.set('n', '<esc>', '<cmd>RegexplainerHide<cr>')
```

**Comment by tomoakley** [comment]
> would this do it?
> 
> ```lua
> vim.keymap.set('n', '<esc>', '<cmd>RegexplainerHide<cr>')
> ```

yeah that's good enough, thanks!
